be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -58,23 +60,12 @@ SinkBuffer::~SinkBuffer() {
 
     DCHECK(is_finished());
 
-    for (auto& [_, buffer] : _buffers) {
-        while (!buffer.empty()) {
-            auto& request = buffer.front();
-            // Once the request is added to SinkBuffer, its ownership will also be transferred,
-            // so SinkBuffer needs to be responsible for the release of resources
-            request.params->release_finst_id();
```

Comment:
will the `_buffers.clear()` call `release_first_id()` ?

Reply:
sure

Reply:
release_finst_id means release the owner ship for first_id. It only used when we call set_allocated_xxx to avoid call delete.
now finst_id was allocated by Message::arena (Message internal memory pool). So we don't have to call release_xxx